.env-example

```diff
@@ -1,10 +1,27 @@
-# Create a `.env` file that specifies invest and python paths
-# on your local system. These variables are set by nodejs on npm start
-# (see package.json scripts)
+# Create a `.env` file in the project root that looks similar to this one.
+# This file is loaded by main.js if it is launched in dev mode. 
 
-INVEST=C:\\Users\\dmf\\Miniconda3\\envs\\invest-py37x\\Scripts\\invest.exe
-PYTHON=C:\\Users\\dmf\\Miniconda3\\envs\\invest-py37x\\python
+# creating this DEVMODE variable allows the renderer process
+# to know whether we're in dev or production mode. Otherwise,
+# only the main process is aware of that. See index.js for use
+# of this variable in the renderer process.
 
+DEVMODE="true"
+
+# It's best to define the invest executeable path here, having it on your 
+# system's path already might work, but might not.
+
+INVEST="C:\\Users\\dmf\\Miniconda3\\envs\\invest-py37x\\Scripts\\invest.exe"
```

Comment:
For this file and line particularly, will this somehow be set by wherever the user Downloads InVEST to? Just curious how this file might look for packaging and deployment.

Reply:
Good question, and part of larger discussion related to #1 . 

Right now, the `.env` file actually only loads in dev mode (i.e. the app is launched via `npm start`), not production mode. But I think it's a valid solution for production as well -- and we have no other solution for this yet, So we could generate and package a `.env` like this with the correct parameters set, depending on OS we're building for.